Instagram/Views/PostTableViewCell.m

```diff
@@ -0,0 +1,44 @@
+//
+//  PostTableViewCell.m
+//  Instagram
+//
+//  Created by Mai Ngo on 6/27/22.
+//
+
+#import "PostTableViewCell.h"
+
+@implementation PostTableViewCell
+
+NSString static *const likeArrayKey = @"likeArray";
+NSString static *const imageKey = @"image";
+NSString static *const usernameKey = @"username";
+
+-(void)setPost:(Post *)post {
+    _post = post;
+    self.postImage.file = post[imageKey];
+    [self.postImage loadInBackground];
+}
+- (IBAction)didTapLike:(id)sender {
+    PFUser *user = [PFUser currentUser];
+    NSArray *likeArray = [[NSArray alloc] init];
```

Comment:
You don't need to call alloc init here since you are overwriting it in the next line. Just: `NSArray *likeArray = [self.post objectForKey:likeArrayKey];`